### PR TITLE
chore(ci): faster transport E2E

### PR DESCRIPTION
**Pull request by Nodonisko**
This should make transport E2E about ~4 minutes faster.

## Related Issue

Resolve

### CODE REVIEW

File: packages/transport/package.json [hunk 1]
```diff
@@ -52,6 +52,8 @@
         "prepublish": "yarn tsx ../../scripts/prepublish.js"
     },
     "devDependencies": {
+        "@babel/preset-env": "^7.23.9",
+        "@babel/preset-typescript": "^7.23.3",
```
Comment (karliatto) [inline packages/transport/package.json:56]: Aren't those dependencies used in jest.config? https://github.com/trezor/trezor-suite/blob/develop/packages/transport/e2e/jest.config.ts#L12

File: packages/trezor-user-env-link/package.json [hunk 2]
```diff
@@ -11,5 +11,8 @@
     "dependencies": {
         "@trezor/utils": "workspace:*",
         "ws": "^8.16.0"
+    },
+    "devDependencies": {
+        "@types/ws": "^8.5.5"
```
Comment (karliatto) [inline packages/trezor-user-env-link/package.json:16]: Is this change related to this PR? :thinking:

File: packages/transport/package.json (hunk 1 above)
Comment (Nodonisko) [reply packages/transport/package.json:56]: Yes, they need to be present in dev dependecies because we are now using `yarn focus` instead of yarn install

File: packages/trezor-user-env-link/package.json (hunk 2 above)
Comment (Nodonisko) [reply packages/trezor-user-env-link/package.json:16]: Yes, this types wasn't installed because of `yarn focus` so they need to be explicitelly added.

**Review by karliatto — APPROVED** [review]
LGTM
